examples/src/main.rs

```diff
@@ -92,15 +92,21 @@ fn main() {
 
     let proof_bytes = proof.to_bytes();
     println!("Proof size: {:.1} KB", proof_bytes.len() as f64 / 1024f64);
-    let conjectured_security_level = options.get_proof_security_level(&proof, true);
+    let conjectured_security_level = options.get_proof_security_level_conjectured(&proof);
```

Comment:
Not from this PR, but I think this line should move into code block which starts on line 99 and ends can also get rid of lines 112 and 113, right?

Reply:
I think we want to print only conjectured security estimates in `std` and both estimates in `no-std`, so I think the code is correct